package.json

```diff
@@ -2,12 +2,13 @@
   "name": "libxmljs",
   "author": "Marco Rogers",
   "contributors": [
-    "Jeff Smick"
+    "Jeff Smick",
+    "Martin von Gagern <Martin.vGagern@gmx.net>"
```

Comment:
All due respect, but if somebody makes substantial contributions, then they should be listed in the package.json as a contributor. It seems just a little bit petty to insist his name should be kept off.